v3.2/glfw/glfw/src/wl_window.c

```diff
@@ -24,6 +24,8 @@
 //
 //========================================================================
 
+#define _GNU_SOURCE
+
```

Comment:
Note that per https://github.com/go-gl/glfw/pull/283#issuecomment-723660957, this will introduce a double definition warning. From having looked into it before, it seems the best solution is to put `wl_window.c` into its own compilation unit and arrange that it isn't built with -D_GNU_SOURCE.

Reply:
Scratch that. Seperate compilation unit doesn't look straightforward. I've just `#undef _GNU_SOURCE` immediately before the `#include "wl_window.c"` instead.